torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1550,24 +1583,23 @@ void insertQuantDeQuantCall(
     quant = g->create(at::Symbol::aten(quantize_func), inputs);
     quant->output()->setDebugName(v->debugName() + ".quant");
     g->insertNode(quant);
-  }
-  Value* original_val = observer->input(1);
-  v->replaceAllUsesWith(original_val);
 
-  // two passes to insert the dequant for every usage
-  // in first pass, identify all the nodes using original observed value.
-  std::vector<Use> uses;
-  for (const auto& use : original_val->uses()) {
-    // Skip quant node and observer node (we need to keep
-    // observer nodes around since we need them to
-    // find the quantization parameters)
-    if (use.user != quant && use.user != observer &&
-        use.user != choose_qparams) {
-      uses.push_back(use);
+    v->replaceAllUsesWith(original_val);
+    // two passes to insert the dequant for every usage
+    // in first pass, identify all the nodes using original observed value.
+    std::vector<Use> uses;
+    for (const auto& use : original_val->uses()) {
+      // Skip quant node and observer node (we need to keep
+      // observer nodes around since we need them to
+      // find the quantization parameters)
+      if (use.user != quant && use.user != observer &&
+          use.user != choose_qparams) {
+        uses.push_back(use);
+      }
     }
+    // in second pass, replace the original observed value with dequant output
+    insertDeQuantForAllUse(g, quant->output(), original_val, uses);
```

Comment:
we don't need to insert dequant for all use here anymore since we have replicate dequant pass in the end.

Reply:
Sounds good. I was wondering if I should remove this